flake.nix

```diff
@@ -967,7 +967,7 @@
                     rpc-file-mode=0660
                     log-timestamps=false
 
-                    plugin=${craneBuildNative.ln-gateway-pkgs}/bin/ln_gateway
+                    plugin=${craneBuildNative.gateway-pkgs}/bin/ln_gateway
```

Comment:
You're running a plugin that doesn't exist here. This is what https://github.com/fedimint/fedimint/issues/1873 is about.

Reply:
Oh I see now.